src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -291,7 +291,7 @@ def get(self, request: Request, organization: Organization | RpcOrganization) ->
                 for ethreshold in category_thresholds:
                     # TODO: filter by environment as well?
                     is_healthy = is_error_count_healthy(ethreshold, error_counts)
-                    ethreshold["is_healthy"] = is_healthy
+                    ethreshold.update({"is_healthy": is_healthy})
```

Comment:
is `is_healthy` always set before this?

Reply:
Yea - I initialize it to `False` up above on line 252